aether-couchdb-sync-module/conf/check_vars.sh

```diff
@@ -38,3 +38,6 @@ check_variable $AETHER_KERNEL_TOKEN "Aether KERNEL token (AETHER_KERNEL_TOKEN)"
 
 # Google ID
 check_variable $GOOGLE_CLIENT_ID    "Google Client ID (GOOGLE_CLIENT_ID)"
+
+# set default value for DEBUG if missing
+DEBUG="$DEBUG"
```

Comment:
What does this line do? As far as I can tell, it wouldn't have any effect. Did you intend to include a default value and export it somehow?

Reply:
Oops!